controller/engine_controller.go

```diff
@@ -1541,14 +1541,18 @@ func (ec *EngineController) startRebuilding(e *longhorn.Engine, replica, addr st
 		}
 
 		// start rebuild
+		sizeRequested, err := ec.ds.IsEngineImageCLIAPIVersionEqualToOrLargerThan(e.Spec.EngineImage, 6)
+		if err != nil {
```

Comment:
Should log error.

Reply:
Resubmit the PR. Please help review, thank you. :)